src/drivers/dshot/module.yaml

```diff
@@ -0,0 +1,54 @@
+module_name: DShot Driver
+serial_config:
+    - command: dshot telemetry ${SERIAL_DEV}
+      port_config_param:
+        name: DSHOT_TEL_CFG
+        group: DShot
+
+parameters:
+    - group: DShot
+      definitions:
+        DSHOT_CONFIG:
+            description:
+                short: Configure DShot
+                long: |
+                    This enabled/disabled DShot. The different modes define different
```

Comment:
enables/disables (i.e. use correct tense)

Reply:
Such an obvious mistake... and I was even consistent.